target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
I replaced my bootmon by my little piece of code containing the jump, and...

```
### RS353aw (Hardware-Rev. 1.0, Firmware-Rev. 1.0) ###

CPU   Check ... passed (MIPS 34KEc [MT] @ 500/250.0 MHz)
SDRAM Check ................................................................................................................................. passed (128 MByte)
FLASH Check  passed (32 MByte)

### Selftest passed ###
Disabling WDT


OpenWrt kernel loader for MIPS based SoC
Copyright (C) 2011 Gabor Juhos <juhosg@openwrt.org>
Decompressing kernel... done!
Starting kernel at 80002000...

[    0.000000] Linux version 4.4.14 (qa@serveurQA) (gcc version 5.3.0 (OpenWrt GCC 5.3.0 50020) ) #62 SMP Wed Jan 11 21:10:30 UTC 2017
[    0.000000] SoC: xRX200 rev 1.2
```
It works !

No CRC anymore, no problem, just boot :)